wazo_confgend/generators/tests/test_extensionsconf.py

```diff
@@ -144,7 +146,7 @@ def test_generate(self, mock_asterisk_conf_dao, mock_ivr_dao):
 
         mock_ivr_dao.find_all_by.return_value = [
             IVR(id=42, name='foo', menu_sound='hello-world'),
-            IVR(id=43, name='bar', menu_sound='youhou'),
+            IVR(id=43, name='bar', menu_sound='youho'),
```

Comment:
Errr... Was that on purpose? :sweat_smile:

Reply:
After checking, this value is not really useful in the test... So I'll make the change (I have it right in front of me).

Reply:
Sorry that is already fixed. Merging...

Reply:
Haha, no it wasn't on purpose, but yeah, it was fixed in the following commit. Good catch though! Thanks! :)